test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl

```diff
@@ -276,4 +276,10 @@ inputs = {
 
   grafana_agent_deploy = true
 
+  # --------------------------------------------------
+  # External Secrets
+  # --------------------------------------------------
+  external_secrets_deploy = true
+  external_secrets_helm_chart_version = "0.9.13"
```

Comment:
Thanks, done

Reply:
Maybe we can go ahead and try the 1.x.x with our terraform that generated flux manifests, actually. But in any case I would do it separately rather than changing how we do things in this PR

Reply:
Find with me. Its probably documented nowhere, but we need to version lock grafana_agent and kube-promtheus-stack until we have gotten rid of kube-promtheus-stack,because they share the same CRDs.